README.md

```diff
@@ -23,3 +23,36 @@ Apache header:
     See the License for the specific language governing permissions and
     limitations under the License.
 
+This application provides an audio interface for conducting tests.
+
+
+This sample [Dropwizard](http://dropwizard.io/) application runs on Google Cloud Platform's
```

Comment:
This is sample application for now, but won't be the same in future. So can you make the change.

Reply:
Removed the word sample from the line in the new commit.